app/api/threesixty.py

```diff
@@ -0,0 +1,144 @@
+from app import token
+from app import mongo
+from app.util import serialize_doc
+from flask import (
+    Blueprint,jsonify, abort, request
+)
+import datetime
+from flask_jwt_extended import (jwt_required, get_current_user
+)
+
+
+bp = Blueprint('threesixty', __name__, url_prefix='/')
+
+#Api for get juniors managers.
+@bp.route('/get_managers', methods=['GET'])
+@jwt_required
+def get_managers():
```

Comment:
hmm. does this api already exist. this look like something which would be used in dashboard as well??

this code should be reused as this is used in multiple places

Reply:
No this is only single API for getting juniors managers. in the dashboard, frontend is showing all details from profile. and all other api is to find managers juniors.